packages/ace-report/package.json

```diff
@@ -17,6 +17,7 @@
   "license": "MIT",
```

Comment:
same here as for `ace-cli`: since we're updating this package, shouldn't we update its version?

Reply:
yes, ace-core too, and ace-http

Reply:
Done https://github.com/daisy/ace/pull/226/commits/34936f751ec56a16fc2ef925f682f191c7d390e1